test/integration/proposal-checks/IntegrationTestBadDebtFlows.sol

```diff
@@ -0,0 +1,371 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.13;
+
+import "@forge-std/Test.sol";
+
+import {PostProposalCheckFixture} from "@test/integration/proposal-checks/PostProposalCheckFixture.sol";
+
+contract IntegrationTestBadDebtFlows is PostProposalCheckFixture {
+    function testVoteForSDAIGauge() public {
+        uint256 mintAmount = governor.quorum(0);
+        /// setup
+        vm.prank(addresses.mainnet("TEAM_MULTISIG"));
+        rateLimitedGuildMinter.mint(address(this), mintAmount);
+        guild.delegate(address(this));
+
+        assertTrue(guild.isGauge(address(term)));
+        assertEq(guild.numGauges(), 1);
+        assertEq(guild.numLiveGauges(), 1);
```

Comment:
These will start to revert after the protocol is live

If you want to vote for a gauge, you can define in state `term = LendingTerm(guild.liveGauges()[0])` and just use the arbitrary first live term for everything